selenium/DockerfileSelenium

```diff
@@ -0,0 +1,15 @@
+FROM python:3.11-bullseye
+
+WORKDIR /selenium
+COPY requirements.txt /selenium/
+RUN pip install -r requirements.txt
+
+COPY . /selenium/
+
+# Install Google Chrome
+RUN wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add -
+RUN sh -c 'echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google-chrome.list'
+RUN apt-get -y update
+RUN apt-get install -y google-chrome-stable
+
+ENV DISPLAY=:99
```

Comment:
```suggestion
# Set display port as an environment variable to avoid crash
ENV DISPLAY=:99
```

Not sure if adding Selenium is still thought about but apparently this line sets the display port to avoid a crash.

Sources:
https://stackoverflow.com/a/67584076/4496102
https://stackoverflow.com/a/51266278/4496102